webproject/src/main/webapp/WEB-INF/web.xml

```diff
@@ -23,7 +23,7 @@
   <!-- Listings Servlet -->
   <servlet>
     <servlet-name>Listings</servlet-name>
-    <servlet-class>/listings.html</servlet-class>
+    <jsp-file>/listings.html</jsp-file>
   </servlet>
```

Comment:
I have revamped the entire web.xml file so there might be a merge conflict here if my PR gets merged first. You can refer to my PR (or main code if my PR gets merged) to copy and paste the new file structure. Otherwise, the conflicts can probably be resolved using GItHub's text editor when you attempt to merge this PR (maybe doing that will be easier).

Reply:
okay!